torchvision/models/optical_flow/raft.py

```diff
@@ -475,6 +475,8 @@ def forward(self, image1, image2, num_flow_updates: int = 12):
             raise ValueError(f"input images should have the same shape, instead got ({h}, {w}) != {image2.shape[-2:]}")
         if not (h % 8 == 0) and (w % 8 == 0):
             raise ValueError(f"input image H and W should be divisible by 8, instead got {h} (h) and {w} (w)")
+        if not (h < 128) and (w < 128):
+            raise ValueError(f"input image H and W should be equal or larger than 128, instead got {h} (h) and {w} (w)")
```

Comment:
This is not going to raise an error if `h < 128` - I assume this is not what we want?

```suggestion
        if h < 128 or w < 128:
            raise ValueError(f"input image H and W should be equal to or larger than 128, instead got {h} (h) and {w} (w)")
```

Reply:
Resolved, thanks!